projects/storefrontapp-cds-integration/src/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Since we have pwa support for this shell app, we should also add a script to run it, like this one: https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/develop/package.json#L49